cms/templates/404.html

```diff
@@ -19,9 +19,9 @@ <h1 class="title title-1">${_("Page not found")}</h1>
       ${_('The page that you were looking for was not found.')}
       ${Text(_('Go back to the {homepage} or let us know about any pages that may have been moved at {email}.')).format(
         homepage=HTML('<a href="/">homepage</a>'),
-        email=HTML('<a href="mailto:{address}">{address}</a>'.format(
+        email=HTML('<a href="mailto:{address}">{address}</a>').format(
```

Comment:
Reviewers: This and some other template fixes below were added to get to 0 violations for certain linter rules.